tests/ci/cdk/cdk/components.py

```diff
@@ -37,11 +37,28 @@ def __init__(self, scope: Construct, id: str, *, project: codebuild.IProject) ->
                                     "codebuild:BatchGetBuildBatches",
                                     "codebuild:ListBuildBatchesForProject",
                                     "codebuild:StopBuildBatch",
+                                    "ec2:TerminateInstances",
                                 ],
                                 resources=[project.project_arn]))
 
+        if ec2_permissions:
+            lambda_function.add_to_role_policy(
+                iam.PolicyStatement(effect=iam.Effect.ALLOW,
+                                actions=[
+                                    "ec2:TerminateInstances",
+                                ],
+                                resources=["arn:aws:ec2:{}:{}:instance/*".format(AWS_REGION, AWS_ACCOUNT)]))
```

Comment:
Can this be further scoped down to only allow terminating some instances based on the tag https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_examples_ec2-start-stop-tags.html.

Reply:
Took me a while, but I was tripping over a self inflicted bug with the wrong tag name... I have the policy working now.